dima.py

```diff
@@ -0,0 +1,64 @@
+from datetime import date 
+ 
+class Medicine: 
+    def __init__(self, name, price, quantity, prescription_needed, expiration_date): 
+        self.name = name 
+        self.price = price 
+        self.quantity = quantity 
+        self.prescription_needed = prescription_needed 
+        self.expiration_date = expiration_date 
+ 
+    def __str__(self): 
+        return f"{self.name} - Price: {self.price}, Quantity: {self.quantity}, Prescription Needed: {self.prescription_needed}, Expiration Date: {self.expiration_date}" 
```

Comment:
Use f-string for data output: replace __str__ method in Medicine class with f-string for better output format.

Reply:
i all fixed